donkeycar/management/base.py

```diff
@@ -196,20 +197,40 @@ def parse_args(self, args):
 
     def run(self, args):
         args = self.parse_args(args)
-        channel = int(args.channel)
 
         if args.arduino == True:
             from donkeycar.parts.actuator import ArduinoFirmata
 
+            channel = int(args.channel)
             arduino_controller = ArduinoFirmata(servo_pin=channel)
             print('init Arduino PWM on pin %d' %(channel))
             input_prompt = "Enter a PWM setting to test ('q' for quit) (0-180): "
+
+        elif args.pwm_pin is not None:
+            from donkeycar.parts.actuator import PulseController
+            from donkeycar.parts import pins
+
+            pwm_pin = None
+            try:
+                pwm_pin = pins.pwm_pin_by_id(args.pwm_pin)
+            except ValueError as e:
+                print(e)
+                print("See pins.py for a description of pin specification strings.")
+                exit(-1)
+            print('init pin {}'.format(args.pwm_pin))
```

Comment:
We moved to f-strings instead of the `format` based strings.

Reply:
I've changes .format() instances to f-strings

Reply:
great